src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTiming.java

```diff
@@ -242,7 +245,7 @@ public static GenericStatus computeChunkStatus2(@Nonnull WorkflowRun run,
         if (exec == null) {
             return null;
         }
-        if (!NotExecutedNodeAction.isExecuted(lastNode)) {
+        if (!NotExecutedNodeAction.isExecuted(lastNode) || checkStageSkippedForConditional(firstNode)) {
```

Comment:
*SHOULD* just be able to look for a TagsAction that is not null and matches the hardcoded skipped string.

Reply:
Yes, done in 08b0e6c